assets/js/feature-tours/unified-dashboard.js

```diff
@@ -68,6 +73,28 @@ const unifiedDashboard = {
 			placement: 'top',
 		},
 	],
+	callback: ( data ) => {
+		const { type, index } = data;
+
+		/*
+		 * The third step of the feature tour is positioned to the bottom on smaller screens.
+		 * Thus, we make sure the tooltip is fully visible by using scrollIntoView.
+		 */
+		if (
+			! [ EVENTS.STEP_BEFORE, EVENTS.STEP_AFTER ].includes( type ) ||
+			index !== 2
+		) {
+			return;
+		}
+
+		const tooltipElement = global.document.querySelector(
+			'.googlesitekit-tour-tooltip'
+		);
+		tooltipElement?.scrollIntoView( {
+			behaviour: 'smooth',
+			block: 'start',
```

Comment:
Also this should be `end` to make sure we scroll to the bottom of the tooltip.
```suggestion
			block: 'end',
```

Reply:
@eugene-manuilov The tooltip is [positioned top](https://github.com/google/site-kit-wp/blob/develop/assets/js/feature-tours/unified-dashboard.js#L68) so it'll need to be top aligned 🤔